src/test/java/com/datapipe/jenkins/vault/it/VaultConfigurationIT.java

```diff
@@ -43,12 +37,16 @@
 import java.util.List;
 import java.util.Map;
 
-import hudson.model.FreeStyleBuild;
-import hudson.model.FreeStyleProject;
-import hudson.model.Result;
-import hudson.tasks.Shell;
-import hudson.util.Secret;
-import jenkins.model.GlobalConfiguration;
+import static org.hamcrest.MatcherAssert.assertThat;
```

Comment:
Please don't mess with import order.

I will have a PR up shortly that fixes import/code style.